aidants_connect_web/templates/aidants_connect_web/id_provider/authorize.html

```diff
@@ -20,13 +21,30 @@ <h1 id="welcome_aidant">Bienvenue sur votre Espace Aidants Connect, {{ aidant.fi
       <h2>Sélectionnez l'usager que vous souhaitez FranceConnecter</h2>
       <p id="instructions">Seuls les usagers avec un mandat en cours sont affichés ici.</p>
       {% if usagers %}
+        {% include "aidants_connect_web/users_search_bar.html" %}
         <fieldset>
           <div id="usagers" class="grid">
             {% for usager in usagers %}
-              <div id="usager" class="tile">
-                <input id="button-{{ usager.id }}" type="submit" value="{{ usager.id }}" name="chosen_usager" />
-                <label id="label-usager" for="button-{{ usager.id }}">
-                  <h3>{{ usager.given_name }} {{ usager.family_name }}</h3>
+              <div
+                class="tile usager"
+                data-controller="filter"
+                data-filter-firstname-value="{% if usager.preferred_username %}{{ usager.preferred_username }}&nbsp;{% endif %}{{ usager.family_name }}"
+                data-filter-familyname-value="{{ usager.given_name }}"
+                data-filter-search-value=""
+                data-filter-irrelevant-result-class="irrelevant-result"
+              >
+                <input
+                  id="button-{{ usager.id }}"
+                  type="submit"
+                  value="{{ usager.id }}"
+                  name="chosen_usager"
+                />
+                <label
+                  id="label-usager-{{ usager.id }}"
+                  class="label-usager"
+                  for="button-{{ usager.id }}"
+                >
+                  {{ usager.given_name }} {{ usager.family_name }}
```

Comment:
Même si on n'a toujours pas le nom d'usage en prod on peut déjà en préparer l'affichage :
 
```suggestion
                  {{ usager.given_name }}
                  {% if usager.preferred_username %}{{ usager.preferred_username }} né(e){% endif %}
                  {{ usager.family_name }}
```

Reply:
(à ne pas commiter tel quel, je l'ai tapé dans l'éditeur de github)